docs/topic-guides/swift/suggest.md

```diff
@@ -0,0 +1,109 @@
+# Suggest
+
+The API for the `SuggestStore` can be found in the [MozillaComponents Swift documentation](https://mozilla.github.io/application-services/swift/Classes/SuggestStore.html).
+
+> The `SuggestStore` is a synchronous, which needs to be wrapped in the asynchronous primitive of the target language you are using it in.
+
+## Setting up the store
+
+You need to import the following primitives to work with the `SuggestStore` (these come from the generated `suggest.swift` file, produced by `uniffi`):
+
+```swift
+import class MozillaAppServices.SuggestStore
+import class MozillaAppServices.SuggestStoreBuilder
+import class MozillaAppServices.Viaduct
+import enum MozillaAppServices.SuggestionProvider
+import enum MozillaAppServices.RemoteSettingsServer
+import struct MozillaAppServices.SuggestIngestionConstraints
+import struct MozillaAppServices.SuggestionQuery
+```
+
+On start up of your application, you create a `SuggestStore` (as a singleton). You do this via the `SuggestStoreBuilder`, which returns a `SuggestStore`:
+
+```swift
+let store: SuggestStore
+
+var builder = SuggestStoreBuilder()
+    .dataPath(path: dataPath)
+
+if let remoteSettingsServer {
+    builder = builder.remoteSettingsServer(server: remoteSettingsServer)
+}
+
+store = try builder.build()
+```
+
+In addition to the `dataPath`, which is the path (the SQLite location) where you store your suggestions, you need a `remoteSettingsServer`, which is the URL to the instance of `RemoteSettings` you will be using (e.g. `https://remote-settings-dev.allizom.org`)
+
+## Ingesting suggestions
+
+We ensure that the Rust networking stack has been initialized via:
+
+```Swift
+import class MozillaAppServices.Viaduct
+
+Viaduct.shared.useReqwestBackend()
+```
```

Comment:
Viaduct initialization is required for all platforms and is required for multiple components.  How about:
 - Creating a viaduct section for this.
 - Any components that require viaduct can say "make sure you initialize viaduct" and link to the section.
 - Expanding this description to say that there are 3 supported options:
    - Any `libxul` based can ignore initialization, since it's handled by `libxul`
    - Using the reqwest backend, which uses the `reqwest` library and a reqwest-managed thread.
    - Implementing the C FFI like `libxul` does (https://searchfox.org/mozilla-central/source/toolkit/components/viaduct).

Does this seem correct @linabutler ?